src/main/webapp/i18n/de/exam.json

```
@@ -678,7 +678,8 @@
         },
         "examTimeline": {
             "title": "Klausurverlauf von {{ username }}",
-            "breadcrumb": "Klausurverlauf"
+            "breadcrumb": "Klausurverlauf",
+            "disabledTooltip": "Der Klausurverlauf ist erst verfügbar, wenn die Klausur abgegeben wurde."
```

Is this warning meant to explain why it is not enabled during the conduction or also after the exam when the student did not submit their exam? Depending on that, this (or something similar) might fit better:
```suggestion
            "disabledTooltip": "Der Klausurverlauf ist nur verfügbar, wenn die Klausur abgegeben wurde."
```